omnibus/config/projects/chef.rb

```diff
@@ -57,7 +57,7 @@
 
 # Chef Release version pinning
 override :chef, version: "local_source"
-override :ohai, version: "master"
+override :ohai, version: "8.10.0"
```

Comment:
This should be `v8.10.0` because that is what the [tag](https://github.com/chef/ohai/tree/v8.10.0) is named